gitoid/src/bin/gitoid.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
Did you intend to leave this file in?

Reply:
Ah, it can be removed. Mostly it was a placeholder so I don't forget to go back and implement a CLI.

Reply:
Not sure if this is what you are thinking of, but I've been playing with a CLI for this here https://github.com/nellshamrell/gitbom-rs-cli. I created this mainly to exercise this library as we developed it and am certainly not attached to it being where it is.

(We had discussed whether it made sense to include the cli in this crate or to keep it separate - I'm definitely open to considering including it within this crate if that would make more sense)

Reply:
Alright, cool. I'll take a look, and we can have a separate discussion later. For now, I'll remove the empty binary crate.